app.js

```diff
@@ -143,6 +143,8 @@ app.get('/map', function (req, res) {
 
 app.get('/twitter-sign-in', controllers.twitter_sign_in.get)
```

Comment:
When this path is accessed, does it use Auth0 or the old Twitter OAuth?

The reason why we wanted to keep both code paths at the same time is so that we can easily toggle between one or the other in case we run into any issues with Auth0 in production. We can toggle between Auth0 or Twitter OAuth via a config setting without needing to revert the codebase.

This is why I was pointing at this [config variable](https://github.com/streetmix/streetmix/blob/master/config/default.js#L55) which toggles between using Transifex (our localization platform) between production and other environments. We should have a config variable like this which tells streetmix to use Auth0 instead of Twitter OAuth. When this variable is `true`, use Auth0. When this variable is `false`, use Twitter OAuth.

Reply:
> When this path is accessed, does it use Auth0 or the old Twitter OAuth?
Old Twitter

Reply:
So basically, this means setting the config variable like localization and have a logic in the front end `MenuBar.jsx` that allows url `prop` to be set (in order to for `sign in` to be an anchor tag and use old twitter) when the config variable `use_auth0`  is `false`?

Reply:
Yep!